docs/pages/versions/unversioned/guides/using-sentry.md

```diff
@@ -53,6 +53,7 @@ Sentry.config('your Public DSN goes here').install();
         {
           "file": "sentry-expo/upload-sourcemaps",
           "config": {
+            "url": "your sentry url here",
```

Comment:
i believe this is only necessary when self-hosting sentry, we should be sure to clarify that this is optional